spec/models/project_spec.rb

```diff
@@ -293,17 +293,39 @@
   end
 
   describe '.goal_between' do
-    let(:start_at) { 100 }
-    let(:ends_at) { 200 }
-    subject { Project.goal_between(start_at, ends_at).order(:id) }
+    let(:start_at)                 { 200 }
+    let(:ends_at)                  { 400 }
+    let!(:all_projects)            { projects_under_start_at + projects_over_ends_at + projects_goal_between }
+    let!(:projects_under_start_at) { create_list(:project, 3, goal: 199) }
+    let!(:projects_over_ends_at)   { create_list(:project, 3, goal: 401) }
+    let!(:projects_goal_between)   {
```

Comment:
I think you don't need the `bang` (`!`) on these ones, since the `all_projects` call everbody.